content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -522,6 +525,7 @@ GET / HTTP/1.1
 X-Remote-User: fido
 X-Remote-Group: dogs
 X-Remote-Group: dachshunds
+X-Remote-Extra-Resource%2Fpath: foo/bar/baz
```

Comment:
a more realistic example might be a namespaced key/value pair: `X-Remote-Extra-acme.com%2Fproject: some-project`

Reply:
done